web/src/components/datasets/DatasetDetailPage.tsx

```diff
@@ -147,9 +170,31 @@ const DatasetDetailPage: FunctionComponent<IProps> = props => {
               />
             </Tabs>
           </Box>
-          <IconButton onClick={() => history.push('/datasets')}>
-            <CloseIcon />
-          </IconButton>
+          <Box display={'flex'} alignItems={'center'}>
+            <Box mr={1}>
+              <Button
+                color='primary'
```

Comment:
Can we make this a `ghost` variant... I think it's a little too strong on the page.

Reply:
Sure, good point